src/matching/matching.jl

```diff
@@ -1,7 +1,7 @@
 __precompile__(true)
 module Matching
 using LightGraphs
-export MatchingResult, maximum_weight_maximal_matching, minimum_weight_perfect_matching
+export MatchingResult, max_weight_matching, maximum_weight_maximal_matching, minimum_weight_perfect_matching
```

Comment:
max_weight_matching->maximum_weight_matching

for coerence with the other functions